run_e2e.sh

```diff
@@ -42,5 +31,7 @@ set -x
     --wait-time-for-ingress-status="${WAIT_FOR_STATUS_TIMEOUT}" \
     --test.timeout="${TEST_TIMEOUT}"
 ret=$?
-set -x
-exit ${ret}
+#set -x
+saveResults
+#exit ${ret}
```

Comment:
do we not want to save the exit code?

Reply:
Not for now. Returning an error breaks the sonobuoy execution. I will add an additional flag to configure this behavior.